.changeset/silent-ducks-wait.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/clerk-react': patch
+'@clerk/types': patch
```

Comment:
No change in functionality so i've marked as patch. Let me know if we think it should be minor though.